docs/installing.adoc

```diff
@@ -293,27 +271,27 @@ $ vagrant box add adb <local path to the downloded image>
 [[set-up-adb-behind-an-http-proxy]]
 == Step 6. (Optional) Set up ADB behind an HTTP proxy
 
-ADB can be set up behind a proxy server. You need to export the proxy
-server information in to the environment and then run `vagrant up`.
+ADB can be set up behind a proxy server. You will need to set a few proxy parameters in the Vagrantfile you chose in the previous step. Refer the https://github.com/projectatomic/vagrant-service-manager#http-proxy-settings[HTTP Proxy Settings] section of the vagrant-service-manager for further details.
 
```

Comment:
Steps to set proxy server information is a very important piece of ADB user experience. So I think this section should should contain VSM config options and we should not redirect users for the information. @Preeticp @budhrg @praveenkumar comments?